build.sbt

```diff
@@ -148,6 +148,13 @@ lazy val httpCore = project("akka-http-core")
   .dependsOn(parsing)
   .addAkkaModuleDependency("akka-stream", "provided")
   .addAkkaModuleDependency("akka-stream-testkit", "test")
+  .addAkkaModuleDependency(
+    "akka-stream",
+    "test",
```

Comment:
So, we compile against a published artifact but then override it here for test compilation / execution.